### PR TITLE
[FIX] Fix for contibutors

### PR DESCRIPTION
**Pull request by mgosai**

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

File: account_payment_credit_card/__manifest__.py
Comment (pedrobaeza) [inline account_payment_credit_card/__manifest__.py]: This key is incorrect. You should put here a GitHub login or remove it.

Comment (max3903) [reply account_payment_credit_card/__manifest__.py]: @pedrobaeza The key for Github logins is `maintainers` ;)
https://github.com/OCA/mis-builder/blob/10.0/mis_builder/__manifest__.py#L40

https://github.com/OCA/OCB/blob/11.0/odoo/addons/base/module/module.py#L254

Comment (pedrobaeza) [reply account_payment_credit_card/__manifest__.py]: OK, then what is the sense of putting a redundant information in a non used key?

Comment (max3903) [reply account_payment_credit_card/__manifest__.py]: @pedrobaeza `maintainer` comes from Odoo and contains a company name. `maintainers` was added by @sbidoul and contains Github logins.

Comment (pedrobaeza) [reply account_payment_credit_card/__manifest__.py]: OK, I won't insist more, but I still think it's a useless key not shown anywhere.

**Comment by max3903** [comment]
@pedrobaeza If that's all it takes to get your approval, happy to remove it :)

**Review by pedrobaeza — APPROVED** [review]
Hehe, thanks for the consideration.

**Pull request merged**
